content/docs/database/postgresql/migrate.md

```diff
@@ -15,26 +15,29 @@ Migrating a PostgreSQL database to Civo involves careful planning to ensure a se
 - Choose a migration time that minimizes downtime and has the least impact on business operations, considering the data volume and migration duration.
 
 ## Prerequisites
-- A Civo MySQL database instance should already be in place with connection details on hand. To create a new database you can follow our getting started guide. 
-- Confirm the availability of the `mysqldump` utility on your system for database copying.
+
+- A Civo Postgres database instance should already be in place with connection details on hand. To create a new database you can follow our getting started guide.
+- Confirm the availability of the `pg_dump` utility on your system for database copying.
```

Comment:
Worth adding a link to https://www.postgresql.org/docs/current/app-pgdump.html or information about where to find the utility?